nip47/notifications/nip47_notifier_test.go

```diff
@@ -175,10 +176,10 @@ func doTestSendNotificationPaymentSent(t *testing.T, svc *tests.TestService, app
 	notifier := NewNip47Notifier(relay, svc.DB, svc.Cfg, svc.Keys, permissionsSvc, transactionsSvc, svc.LNClient)
 	notifier.ConsumeEvent(ctx, receivedEvent)
 
-	assert.NotNil(t, relay.PublishedEvent)
-	assert.NotEmpty(t, relay.PublishedEvent.Content)
+	assert.NotNil(t, relay.PublishedEvents[1])
+	assert.NotEmpty(t, relay.PublishedEvents[1].Content)
```

Comment:
`[1]` because `[0]` is of kind `23196` and it can't be decrypted by v1.0 NIP-44 cipher